package.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "gridicons",
-  "version": "3.2.0",
+  "version": "3.3.0",
   "main": "dist/index.js",
   "files": [
-    "dist/"
+    "dist/",
+    "sources/react/icons-offset.js",
```

Comment:
this one feels a little odd. I think ideally it would be part of `dist/index.js`, or at least live in `dist/`. 

Probably ok for now though... hmmm.

Reply:
Yeah, I think copying it to `dist` would be enough for now. That's where the current docs tell folks to import individual bits from already.

Reply:
No problem! I copied it to `dist/cjs` to distinguish from the other files, which have already been run through a bundler. How does that sound?

Reply:
why cjs? that didn't exist before...

Reply:
The current situation is already a bit problematic, since we have three things under `dist`: `index.js`, with all the icons, `example.js`, with a pre-packaged example with a bunch (maybe all?) icons, and then the individual `*.js` icon files. There's no real distinction between them, and the WebPack rule in Calypso was generating duplicates because of it (now fixed by explicitly excluding `example.js` from the bundle).

I'm reluctant to add yet another special file that lives in the same directory as everything else. Adding a new directory and placing `icons-offset.js` in there seemed like a good option, but `cjs` may indeed be a misleading name, since the other files are exposed as CommonJS modules too, now that I look closer. Would e.g. `util` be better?

Reply:
Here's what we currently ship in `dist/`, for reference:
https://unpkg.com/gridicons@3.2.0/dist/

Reply:
I think `dist/util` would be a good spot. Agreed that this package needs a rework.

Reply:
Done! 👍